pkg/ibmvpc/examples/out/lint_out/basic_sg1_Lint

```diff
@@ -0,0 +1,8 @@
+"Firewall rules implying different connectivity for different endpoints within a subnet" issues:
+------------------------------------------------------------------------------------------------
+In VPC test-vpc1-ky, Network acl sg1-ky rule's indexed 1 splits subnet subnet1-ky (10.240.10.0/24). Splitting rule details: index: 1, direction: inbound,  conns: protocol: all, remote: sg1-ky (10.240.10.4/32), local: 0.0.0.0/0
```

Comment:
`Network acl sg1-ky` should be replaced with `security group sg1-ky` ?

Reply:
obviously.... fixed